src/themes/default/elements/button.variables

```diff
@@ -253,7 +253,8 @@
 /*-------------
    Tertiary
 -------------*/
-@tertiaryLinePadding: 0.5em;
+@tertiaryVerticalPadding: 0.5em;
+@tertiaryHorizontalPadding: 0;
```

Comment:
The default value for horizontal padding should also be 0.5em to keep the same overall padding for the default theme as before
otherwise:
before: `padding: 0.5em;`
after: `padding: 0.5em 0;`

Reply:
@lubber-de Ah yes, my apologies. I just pasted my own preference there. Adjusted now to 0.5em.